### PR TITLE
Use new endpoint

### PR DESCRIPTION
**Pull request by oterral**

### CODE REVIEW

File: src/layers/TrackerLayer.js [hunk 1]
```diff
@@ -163,29 +182,63 @@ class TrackerLayer extends Layer {
   }
 
   style(props) {
-    const { type, name, id, color } = props;
+    const { type, name, id, color, textColor, delay } = props;
+    if (delay > 0) {
+      // console.log(delay, getDelayColor(delay), getDelayText(delay));
```
Comment (edellucien) [inline src/layers/TrackerLayer.js]: could maybe remove this console.log

File: src/layers/TrajservLayerOld.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  }
+
+  updateTrajectories() {
+    this.fetchTrajectories(
+      `${this.url}/trajectories?${this.getUrlParams({
+        attr_det: 1,
+      })}`,
+    ).then(data => {
+      // For debug purpose , display the trajectory
+      // this.olLayer.getSource().clear();
```
Comment (edellucien) [inline src/layers/TrajservLayerOld.js]: can remove

File: src/layers/TrajservLayerOld.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          for (let k = 0; k < path.length; k += 1) {
+            // d: delay. When the train is stopped at a station.
+            const {
+              x,
+              y,
+              a: timeAtPixelInScds,
+              d: delayAtStation,
+              ad: arrivalDelay,
+            } = path[k];
+            // coords.push(this.map.getCoordinateFromPixel([x, y]));
```
Comment (edellucien) [inline src/layers/TrajservLayerOld.js]: can remove?

File: src/layers/TrackerLayer.js (hunk 1 above)
Comment (oterral) [inline src/layers/TrackerLayer.js]:
```suggestion
```

File: src/layers/TrajservLayerOld.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    super({ ...options });
+  }
+
+  updateTrajectories() {
+    this.fetchTrajectories(
+      `${this.url}/trajectories?${this.getUrlParams({
+        attr_det: 1,
+      })}`,
+    ).then(data => {
+      // For debug purpose , display the trajectory
```
Comment (oterral) [inline src/layers/TrajservLayerOld.js]:
```suggestion
```

File: src/layers/TrajservLayerOld.js [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    super({ ...options });
+  }
+
+  updateTrajectories() {
+    this.fetchTrajectories(
+      `${this.url}/trajectories?${this.getUrlParams({
+        attr_det: 1,
+      })}`,
+    ).then(data => {
+      // this.olLayer.getSource().clear();
```
Comment (oterral) [inline src/layers/TrajservLayerOld.js]:
```suggestion
```

File: src/layers/TrajservLayerOld.js [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+          for (let k = 0; k < path.length; k += 1) {
+            // d: delay. When the train is stopped at a station.
+            const {
+              x,
+              y,
+              a: timeAtPixelInScds,
+              d: delayAtStation,
+              ad: arrivalDelay,
+            } = path[k];
+            // coords.push(this.map.getCoordinateFromPixel([x, y]));
```
Comment (oterral) [inline src/layers/TrajservLayerOld.js]:
```suggestion
```

File: src/layers/TrajservLayer.js [hunk 7]
```diff
@@ -219,95 +214,91 @@ class TrajservLayer extends TrackerLayer {
       .then(resp => TrajservLayer.translateTrajStationsResp(resp));
   }
 
-  /**
-   * Fetch trajectories
-   */
-  fetchTrajectories() {
-    if (this.abortController) {
-      this.abortController.abort();
+  getUrlParams(extraParams = {}) {
```
Comment (edellucien) [inline src/layers/TrajservLayer.js]: Would be nice to add a method description for the documentation

**Comment by edellucien** [comment]
Very minor comment added, else good!

File: src/layers/TrajservLayer.js (hunk 7 above)
Comment (edellucien) [reply src/layers/TrajservLayer.js]:
```suggestion
  /**
   * Returns the URL Parameters
   * @param {Object} extraParams
   * @returns {Object}
   */
  getUrlParams(extraParams = {}) {
```

**Pull request merged**